### DISCUSSION TITLE
Improve image loading experience

### DISCUSSION THREAD

**Discussion by alanchan-dev** [avitorio/outstatic#150] [Ideas]
Problem I have:
When uploading an image from file, it stores the image in the asset folder. NextJS image should be able to use static image imports to utilize the blur placeholder but it can't work with the url to the image.

Possible solution:
An option to tick when upload image from file to also generate an additional file with very low quality with an additional url such as blurCoverImage. So can pass the url to Image to show the blur image first.

**Comment by avitorio** [comment]
Hey @alanchan-dev, this is a good suggestion.

If you want, feel free to create a PR to implement your vision for this feature. Let me know if you need any help.
